src/bowlingGame.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+function BowlingGame () {
+  this.rolls = [];
+  this.finished = false;
+  this.numberOfRolls = 0;
+};
+
```

Comment:
This is clever! I hadn't though about feeding in the 'game' as an argument!  Might be me, but took me a little while to work it out so instructions would be helpful.